main.js

```diff
@@ -10,4 +11,288 @@ function closeMenu() {
+
+
+      <div class="popup_button_main_container">
+
+
+        <div class="popup_buttons_container">
+        
+        <a class="popup_buttons" href="https://bilalrajput09.github.io/My-Portfolio/">See Live</a>
+        
+        <span class="popup_icons">
+
+            <img src="./img/see-live.png">
+            
+        </span>
+
+        </div>
+        <div class="popup_buttons_container">
+        
+        <a class="popup_buttons" href="https://github.com/bilalrajput09">See Source</a>
+
+        <span class="popup_icons">
+
+        <img src="./img/github-white-popup.png">
+
+        </span>
+        </div>
+      </div>
+    </div>
+    </div>`;
+  document.body.appendChild(firstProjectContainer);
+  backdrop.classList.remove('backdrop-hidden');
+});
+
+// eslint-disable-next-line no-unused-vars
+function closePopupWindow() {
+  backdrop.classList.add('backdrop-hidden');
+  firstProjectContainer.style.display = 'none';
+}
+
+function createModals(projects) {
+  projects.forEach((project) => {
+    const modal = document.createElement('div');
+    modal.classList.add('main_popup_container');
+    modal.classList.add('modal-hidden');
+    modal.id = `modal-${project.id}`;
+    modal.innerHTML = `
+    <div class="popup_heading_container">
+
+    <h2 class="main_popup_heading">${project.fullName}</h2>
+
+    <div>
+    
+    <img class="close_popup"
+    onclick = "closeModal('modal-${project.id}')"
+    src="./img/cross.png" ">
+    
+    </div>
+
+    </div>
+
+    <ul class="popup_language_box">
+    
+    ${project.technologies.map((tech) => `<li class = "language_box_item" >${tech}</li>`).join('')}
+
+    </ul>
+    <div class="popup_grid_container">
+    
+    <img src="./img/popup-img-desktop.png" class="popup_img">
+
+
+      <p class="popup_para">${project.description}</p>
+
+
+      <div class="popup_button_main_container">
+
+
+        <div class="popup_buttons_container">
+        
+        <a class="popup_buttons" href = "${project.liveLink}">See Live</a>
+        
+        <span
+            class="popup_icons">
+
+            <img src="./img/see-live.png">
+            
+        </span>
+
+        </div>
+        <div class="popup_buttons_container">
+        
+        <a class="popup_buttons" href = "${project.linkToSource}">See Source</a>
+
+        <span class="popup_icons">
+
+        <img src="./img/github-white-popup.png">
+
+        </span>
+        </div>
+      </div>
+    </div>
+    `;
+    document.body.appendChild(modal);
+  });
+}
+
+// eslint-disable-next-line no-unused-vars
+function showModal(btn) {
+  const modal = document.getElementById(btn.dataset.target);
+  modal.classList.remove('modal-hidden');
+  backdrop.classList.remove('backdrop-hidden');
+}
+
+// eslint-disable-next-line no-unused-vars
```

Comment:
Please don't disable linters while trying the fix the issues. They are important to prone code errors and have standardized code 👍